04.class/memo.js

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env node
+
+import readline from "readline";
+import Enquirer from "enquirer";
+import Memo from "./class/memo.js";
+
+const [, , option] = process.argv;
+
+switch (option) {
+  case "-l": {
+    const memo = new Memo();
+    const memos = await memo.selectAll();
+    memos.forEach((memo) => {
+      console.log(memo.content.split(/\n/)[0]);
+    });
+    break;
+  }
+
+  case "-r": {
+    const memo = new Memo();
+    const memos = await memo.selectAll();
+    const choices = memos.map((memo) => {
+      return { name: memo.content.split(/\n/)[0], value: memo.content };
+    });
+    const question = {
+      message: "Choose a memo you want to see:",
+      choices: choices,
```

Comment:
プロパティ名と、その値を指定する変数名が同名のときは、省略記法が使えます。

```suggestion
      choices,
```